npm/pkg/controlplane/translation/translatePolicy.go

```diff
@@ -183,6 +197,8 @@ func podSelectorIPSets(ipSetForSingleVal []string, ipSetNameForMultiVal map[stri
 	return podSelectorIPSets
 }
 
+// targetPodSelectorInfo converts podSelector information to operators and corresponding label information.
+// The label information has various types based on type of labels (e.g., single value or multiple value in labels).
```

Comment:
remove this function since if you replace targetPodSelector() with podSelector()

Reply:
Yes. There will be huge deletions in next PR. So, I would like to clean-up PR in the next one to increase unnecessary load of reviews. is it okay?